core/frontend/src/components/app/SettingsMenu.vue

```diff
@@ -184,6 +238,23 @@ export default Vue.extend({
         })
       this.showDialog(false)
     },
+    async remove_mavlink_log_files(): Promise<void> {
+      await back_axios({
+        url: `${API_URL}/services/remove_mavlink_log`,
+        method: 'post',
+        params: {
+          i_know_what_i_am_doing: true,
+        },
+        timeout: 5000,
```

Comment:
note: the other remove requests are 2 seconds

Reply:
I changed because after some tests seems that removing the tlogs and bins take more time and 2 seconds was generating timeouts. Maybe I'll unify both to 5 seconds